<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeakerRecognition</class>
 <widget class="QMainWindow" name="SpeakerRecognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speaker Recognition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Speaker Identification</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textArea">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSamples">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Utilizza samples di test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFileAudio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Utilizza voci da file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecAudio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Registra nuove voci</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Inizia il test!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewSession">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuova sessione</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SpeakerRecognition, software developed by Alessandro Gerardi - 2019.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string># riconoscimenti = 0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textAreaNomi">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>171</width>
      <height>79</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartTrain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Inizia training</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCrediti">
   <property name="text">
    <string>Crediti</string>
   </property>
  </action>
  <action name="actionEsci">
   <property name="text">
    <string>Esci</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
